### PR TITLE
Jetpack Cloud: Add route to temporarily switch request type

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Add a route that forces Jetpack Cloud to temporarily switch from oauth to proxy-style requests

#### Testing instructions

1. Boot local Jetpack.com
2. Navigate to WordPress.com in a private tab and login
3. in the same tab navigate to `http://jetpack.cloud.localhost:3000/oauth-override`
4. verify you are taken to the `/backup` or `/scan` page for the primary site for the account

### CODE REVIEW

File: client/landing/jetpack-cloud/index.js [hunk 1]
```diff
@@ -7,10 +7,11 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import { translate } from 'i18n-calypso';
+import { getCurrentUser } from 'state/current-user/selectors';
```
Comment (cbauerman) [inline client/landing/jetpack-cloud/index.js:10]: alpha sort

**Comment by ChaosExAnima** [comment]
Would this cause any security issues?

**Comment by mattgawarecki** [comment]
I tested and everything seemed to work as described for me. 👍 

> Would this cause any security issues?

This is a good question, but one I'm not qualified to answer. 😅 Since we're switching to proxy-based authentication and not disabling authentication all together, I'm inclined to think it's okay? I'll hold on approval for now, though.

**Comment by cbauerman** [comment]
> Would this cause any security issues?

I think we are OK, since Calypso uses the the same request and we are fundamentally not changing anything about how either work. Authentication is still all through WordPress.com

**Pull request merged**